backend/de.metas.business/src/main/java/de/metas/payment/api/impl/PaymentBL.java

```diff
@@ -577,6 +581,30 @@ public boolean isCashTrx(@NonNull final I_C_Payment payment)
 		return tenderType.isCash();
 	}
 
+	@Override
+	public void fullyWriteOffPayments(@NonNull final Iterator<I_C_Payment> payments)
+	{
+		while (payments.hasNext())
+		{
+			final I_C_Payment payment = payments.next();
```

Comment:
refactor and introduce fullyWriteOffPayment(I_C_Payment)

Reply:
✔️